policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_disability_income_subtraction.py

```diff
@@ -0,0 +1,35 @@
+from policyengine_us.model_api import *
+
+
+class mt_disability_income_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana disability income subtraction"
+    defined_for = StateCode.MT
+    unit = USD
+    definition_period = YEAR
+    reference = "https://rules.mt.gov/gateway/RuleNo.asp?RN=42%2E15%2E217"
+
+    def formula(tax_unit, period, parameters):
+        # first select head and spouse with ages below the specific threshold, and
+        # select those who are retired on total disability and disabiled, and
+        # calculate the corresbonding disability benefits.
+        person = tax_unit.members
+        p = parameters(
+            period
+        ).gov.states.mt.tax.income.subtractions.disability_income
+        age_eligible_head = tax_unit("age_head", period) < p.age_threshold
```

Comment:
use the following person level variables instead of `age_head` and `age_spouse`
`age` 
`is_tax_unit_head_or_spouse`

this will allow us to shorten the code for one calculation for head and spouse